MedicationScheduler/MedSchedulerApp/templates/home.html

```diff
@@ -1,13 +1,24 @@
+{% extends "base.html" %}
+{% load static %}
+
 {% block content %}
-  <h2>Home Page</h2>
-  <div>
-    {% if user.is_authenticated %}
-    <p>Hello {{user.username}} | {{user.email}}</p>
-    <a href="/logout">Log Out</a>
-    {% else %}
-    <p>You are not Authenticated</p>
-    <a href="/login">Log In</a>
-    <a href="/register">Register</a>
-    {% endif %}
-  </div>
-{% endblock %}
\ No newline at end of file
+  {% if user.is_authenticated %}
```

Comment:
What I was thinking was that the home view would redirect the user to the login page if they weren't logged in. But I'm open to either way.

Reply:
Oh, okay. My idea was that we need a front or "home" page for the app to talk about what our app is for